contracts/strategies/CurveStrategy.sol

```diff
@@ -493,7 +493,6 @@ contract CurveStrategy is Ownable, IStrategy {
     */
     function getAccumulatedRewardTokenAmounts(bool disableRewardTokenClaim)
         external
-        view
```

Comment:
I may have missed something, but why we need to remove the `view` from this method?

Reply:
mentioned below